### ISSUE TITLE
Command line baking crash in my system.

### ISSUE THREAD

**Issue by YUNPERO** [rlguy/Blender-FLIP-Fluids#595]
#### System Information

**Blender Version:** 3.3.0, master, 2022-09-06 15:39, 0759f671ce1f
**Addon Version:** A FLIP Fluid Simulation Tool for Blender (v1.5.0 Stable 07-SEP-2022)
**OS:** Windows-10-10.0.19044-SP0
**GPU:** NVIDIA GeForce RTX 2080 Ti/PCIe/SSE2 NVIDIA Corporation 4.5.0 NVIDIA 516.59
**CPU:** AMD Ryzen 9 3950X 16-Core Processor            
**RAM:** Gskill 32gb 3600

#### Describe the bug

[blender.crash.txt](https://github.com/rlguy/Blender-FLIP-Fluids/files/9542313/blender.crash.txt)

In blender ui, bake cache seems working fine. But when I bake cache through the command line, baking didn't work.

#### Screenshots
![2022-09-11](https://user-images.githubusercontent.com/82076650/189524015-0a63d666-5cbf-4719-9a7a-738a4b8c7582.png)
Worked well in blender Ui.

But,
Command line bake was not. 
![2022-09-11 (1)](https://user-images.githubusercontent.com/82076650/189524050-b6a02485-b36f-48c9-88f0-830da1cba870.png)

**Comment by rlguy** [comment]
Hi, thanks for the report! From the screenshot, there is an error reported from the RDBLab addon. Does disabling the addon, restarting Blender, and trying the command line bake again still result in a crash?

**Comment by YUNPERO** [comment]
> 

Hi! I removed rdblab addon but It didn't work either. I doubted that the cmd function itself was not working but 'Launch Render' in command line tool panel worked fine.

**Comment by rlguy** [comment]
Thanks for the update! I think we will need to try to reproduce this issue on our systems to diagnose the problem. Would you be able to provide the following items:

1. Attach or link to a Blend file that reproduces the issue.
2. Your Blender system info file. This file can be generated with **Blender > Help > Save System Info**
3. Copy the baking command and paste it here
    - ![image](https://user-images.githubusercontent.com/4285191/190001747-f5ad7b8e-79a8-4cf8-b0eb-215e6e56a0a2.png)

Let me know if you have any questions or find more information about the issue.

**Comment by YUNPERO** [comment]
1.
It is just default file only has domain and fluid object. nothing added :)


2.
Blender Version: 3.3.0, master, 2022-09-06 15:39, 0759f671ce1f
Addon Version: A FLIP Fluid Simulation Tool for Blender (v1.5.0 Stable 07-SEP-2022)
OS: Windows-10-10.0.19044-SP0
GPU: NVIDIA GeForce RTX 2080 Ti/PCIe/SSE2 NVIDIA Corporation 4.5.0 NVIDIA 516.59
CPU: AMD Ryzen 9 3950X 16-Core Processor            
RAM: Unknown (fill in)

3.
"C:\Program Files\Blender Foundation\Blender 3.3\blender.exe" --background "C:\Users\YOOM\Desktop\flip\untitled.blend" --python "C:\Users\YOOM\AppData\Roaming\Blender Foundation\Blender\3.3\scripts\addons\flip_fluids_addon\resources\command_line_scripts\run_simulation.py"


And...
I figured it out. I have addon named extreme-pbr. It is the problem of crash. I removed it, and bake is worked!
